tests/http-post-with-binary-response/test.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+
+# disables checksum verification
+args:
+  - -k none
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 1
+  - filter:
+      count: 1
+      match:
+        event_type: fileinfo
+        fileinfo.sha256: "b1dbafe74eb2c8c7f3f696c304132b1ac8837bed9fbbc892fdcf947ba21b6aca"
```

Comment:
Where do these hashes come from ?

There does not seem to be any gap in this pcap. Just a binary response to a regular HTTP POST request, right ?